cmd/insights/contributors.go

```diff
@@ -134,11 +136,29 @@ func (cis contributorsInsightsSlice) BuildOutput(format string) (string, error)
 		return utils.OutputJSON(cis)
 	case constants.OutputYAML:
 		return utils.OutputYAML(cis)
+	case constants.OuputCSV:
+		return cis.OutputCSV()
 	default:
 		return "", fmt.Errorf("unknown output format %s", format)
 	}
 }
 
+func (cis contributorsInsightsSlice) OutputCSV() (string, error) {
+	if len(cis) == 0 {
+		return "", fmt.Errorf("repository is either non-existent or has not been indexed yet")
+	}
+	b := new(bytes.Buffer)
+	writer := csv.NewWriter(b)
+	err := writer.WriteAll([][]string{{"Repository URl", "New Contributors", "Recent Contributors", "Alumni Contributors", "Repeat Contributors"}, {
```

Comment:
> "Repository URI"

Do we usually refer to it as URI or URL? I'm not entirely 100% certain. Looks like in our database tables we use `url`

Reply:
yep, should be URL, that was a typo.